### DISCUSSION TITLE
Unable to drag and drop files to web browser for upload

### DISCUSSION THREAD

**Discussion by Splash-IT** [iterate-ch/cyberduck#14870] [Q&A]
Hello,

when I want to manually transfer files via drag & drop from the CyberDuck window to an upload browser window, it brings up an error with allegedly 0 byte files.
Is this due to CyberDuck or MAC security settings?
If I cache the files on MAC, it works, but it's very cumbersome.

Thanks
Tilo

**Comment by tatze96** [comment]
- Which protocol (e.g. WebDAV, SFTP, S3, etc.) is in use?
- Do you try to copy files between two servers?
- Do you refer to a web browser window or an additional Cyberduck browser window with the term "upload browser window"?

**Comment by Splash-IT** [comment]
Hello,

   - Which protocol (e.g. WebDAV, SFTP, S3, etc.) is in use?
   I use WebDAV via https
   [image: grafik.png]
   - Do you try to copy files between two servers?
   No, betwenn CyberDuck Window and Safari (Synology-Test) Upload Tab
   - Do you refer to a web browser window or an additional Cyberduck
   browser window with the term "upload browser window"?
   [image: grafik.png]
   there will be emtpty
   [image: grafik.png]
   and no file was been uploaded


Thanks
Tilo


*Mit freundlichen Grüßen*
*Tilo Gerlach*
------------------------------

*IN*ternet-*MA*rketing-*CO*mputer
*Inhaber: Tilo Gerlach*
Registergericht: Chemnitz

Falkensteiner Straße 41
D-08239 Trieb / Vogtland

*Telefon:* +49 (0)3746 3226971 <+4937463226971>
*Telefax:* +49 (0)3746 3226972 <+4937463226972>
*Mobilfunk:* *+49 (0)172 376 3321* <+491723763321>
*E-Mail:*
***@***.*** ***@***.***>**Web:*
*Dein-Auftritt.com* <https://dein-auftritt.com/>*DeinAuftritt **✓ Einfach **•
Online **• Erfolgreich*
------------------------------
Diese E-Mail ist nur für den Adressaten bestimmt und kann vertrauliche
und/oder rechtlich geschützte Informationen enthalten. Wenn Sie nicht der
richtige Adressat sind oder diese E-Mail irrtümlich erhalten haben,
informieren Sie bitte sofort den Absender und vernichten Sie diese Mail.
Das unerlaubte Kopieren sowie die unbefugte Weitergabe dieser Mail ist
nicht gestattet.



Am Di., 4. Juli 2023 um 13:39 Uhr schrieb Laura ***@***.***>:

**Reply by tatze96** [reply]
The screenshots aren't visible as they were attached to the email. Any additional files (screenshots, log files, etc.) need to be attached through the GitHub web interface, as they wouldn't be visible to us otherwise.

Do you still see the described behavior within the current version of Cyberduck (8.6.3)?
